specs/clinicalServices/clinicalObservations.cpt

```diff
@@ -96,4 +96,16 @@
 * Verify no error displayed on page
 
 # Doctor adds a drug that has contraindication with the diagnosis <diagnosisName>
-* Doctor add the drug for <diagnosisName>
\ No newline at end of file
+* Doctor add the drug for <diagnosisName>
+
+# Goto ICD Mappings Demonstrator and get the ICD-10 codes for the <diagnosisName>
+* Goto ICD Mappings Demonstrator portal
+* Enter <diagnosisName>
+* Get the ICD-10 code for the SNOMED diagnosis
+
+# Doctor add diagnosis <diagnosis_name> with <mapRule with age condition>
```

Comment:
Where are you using the second parameter?

Reply:
as we are using same function for different condition(like age,gender) ,so to identify in step which condition we are checking I have added this

Reply:
You have defined "Doctor add diagnosis <diagnosis_name> with <mapRule with age condition>". However "mutiple mapGroup", "mapRule with age condition", "mapRule with gender condition" calls the same definition which is not correct